packages/js/src/MatomoTracker.ts

```diff
@@ -335,7 +335,7 @@ class MatomoTracker {
    * @param name The name of the instruction to be executed.
    * @param args The arguments to pass along with the instruction.
    */
-  pushInstruction(name: string, ...args: any[]): MatomoTracker {
+  pushInstruction(name: string, ...args: unknown[]): MatomoTracker {
```

Comment:
Why are we changing these types?

Reply:
Eslint gave warnings about this: we should rather use `never` or `unknown`. In our case we should use `unknown` as we don't know the types that can be passed through